dcat/examples/ga-courts.ttl

```diff
@@ -18,28 +18,28 @@
   rdf:type owl:Ontology ;
   dct:created "2018-09-18" ;
   dct:creator <https://orcid.org/0000-0002-3884-3420> ;
-  rdfs:label "GA Judicial Courts" ;
+  rdfs:label "GA Judicial Courts"@en;
   owl:imports <http://www.w3.org/2006/vcard/ns> ;
   owl:imports <http://www.w3.org/ns/dcat> ;
 .
 ga-courts:GA
   rdf:type v:Organization ;
-  rdfs:label "Commonwealth of Australia (Geoscience Australia)" ;
-  v:country-name "Australia" ;
+  rdfs:label "Commonwealth of Australia (Geoscience Australia)"@en;
+  v:country-name "Australia"@en;

```

Comment:
I don't think the values in a vCard are language-specific strings

Reply:
```suggestion
  v:country-name "Australia" ;
```